benchmarks/inference/llm-bench/src/llm_bench/arg_parsing.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+from .benchmark_runner import BenchmarkConfig
+from .clients import client_config_classes
+
+
+def parse_args_to_configs(args):
+    # TODO: Comment and add type hints
```

Comment:
Nit: Does this TODO still need to be done or can it be removed?

Reply:
Still needs to be done, I'm still updating a few small things - but the major changes are completed